### PR TITLE
Document support for container cluster managers Mesos and Kubernetes.

### PR DESCRIPTION
**Pull request by mattray**
Initial documentation for working with both.

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @davidwrede and @juliandunn to be potential reviewers

File: www/source/docs/kubernetes.html.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Kubernetes with Habitat
+---
+
+[Kubernetes](http://kubernetes.io/) is an open source container cluster manager that is available as a stand-alone platform or embedded in several distributed platforms including [Google's Container Engine](https://cloud.google.com/container-engine/) and [Tectonic](https://tectonic.com/) by [CoreOS](https://coreos.com/). Habitat and Kubernetes are complementary, Kubernetes focuses on providing a platform for deployment, scaling, and operations of application containers across clusters of hosts while Habitat manages the creation and configuration of those application containers.
+
+# Docker and ACI
```
Comment (davidwrede) [inline www/source/docs/kubernetes.html.md]: Should be H2s (##)

File: www/source/docs/kubernetes.html.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Habitat packages are supported in both Docker and ACI formats. Kubernetes currently supports the Docker runtime and integration of the rkt container runtime (an implementation of the App Container spec) is under active development.
+
+# kubectl
```
Comment (davidwrede) [inline www/source/docs/kubernetes.html.md]: Same comment as above.

File: www/source/docs/kubernetes.html.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+       kubectl get pods -l run=mytutorial
+
+## Environment variables and Networking
+
+Kubernetes supports passing [environment variables](http://kubernetes.io/docs/user-guide/environment-guide/) into containers, this needs to be documented for use with Habitat.
+
+## Multi-container Pods
+
+Multi-container pod support through Habitat is still under active development, there may need to be a generated configuration file output from a CLI. Habitat's gossip protocol is on port 9634 and transmitted via UDP, it needs to be available by default to all of the containers in the pod.
```
Comment (juliandunn) [inline www/source/docs/kubernetes.html.md]: comma-splice after "development" as well as after "UDP"

Comment (mattray) [inline www/source/docs/kubernetes.html.md]: fixed

**Comment by juliandunn** [comment]
Replacing with another PR to make sure the filenames are correct.

**Pull request closed without merging**